client/keys/migrate.go

```diff
@@ -62,7 +62,7 @@ func runMigrateCmd(cmd *cobra.Command, args []string) error {
 
 	var (
 		tmpDir   string
-		migrator keyring.InfoImporter
+		migrator keyring.Importer
```

Comment:
i'm not sure why github is showing diff of another PR...

Reply:
I've seen it already in the past.